src/components/Nav.tsx

```diff
@@ -43,7 +43,7 @@ const Nav = () => {
 
   return (
     <nav className={`navbar navbar-expand-lg px-2 py-2 ${poppins.className} border-bottom`}>
-      <div className=""></div>
+      <div></div>
```

Comment:
React official documentation recommends the use of self-closing tags. Refactor to `<div />`